_includes/specials/taproot/en/06-multisignatures.md

```diff
@@ -0,0 +1,147 @@
+something like the following:
+
+1. The wallet for each participant generates a [BIP32][] xpub that is shared
+   with all the other participants (the same as is commonly done now for
+   multisigs)
+
+2. The wallet also generates a set of nonces that are also shared with
+   the other participants.  The wallet can generate these nonces using
+   BIP32 hardened derivation.  Nonces are 32 bytes and you need two of
+   them per signature.  For infrequently used wallets, all the nonces
+   needed for the entire wallet lifetime can be shared up front.  For
+   more frequently used wallets (e.g. LN routing nodes), each wallet can
+   send its signature for the current transaction along with its nonces
+   for the next transaction.
+
+3. Any of the wallets can then generate an aggregated public key by
+   combining its pubkey at a certain BIP32 depth with pubkeys at the
+   same depth from all other wallets in the multisignature association.
+   The aggregated public key can be used to receive P2TR payments.
+
+4. When one of the wallets wants to spend the funds, it uses a
+   [PSBT][topic psbt]-based workflow similar to what it would use with
+   script-based multisig.  Unlike multisig, the wallet uses its next
+   nonces and the next nonces of all the other participants to create a
+   shared nonce according to the MuSig2 algorithm; it then creates a
+   partial signature over that nonce and the transaction.
+
+5. When the other wallets receive the PSBT, they use the same procedure.
+   The partial signatures are then combined to create the final
+   signature and the transaction is broadcast.
+
+### Threshold signing
+
+By themselves, the MuSig family of multisignature schemes only give you
+n-of-n signing---every party who contributes a key towards the
+aggregated public key must also contribute a partial signature to the
+final signature.  That works perfectly well as a direct replacement for
+some uses of script-based multisig today, such as spending 2-of-2 LN
+funding outputs, but it's a departure from other popular policies such
+as the 2-of-3 multisig script used by many exchanges.
+
+Several developers are working [threshold signature][topic threshold
+signature] schemes that will bring the same efficiency and privacy
+benefits of multisignatures to k-of-n scenarios, but there's a simple
+trick that can be used until those schemes are available.
+
+In many threshold cases, it's known in advance which participants are
+most likely to sign.  For example, in a 2-of-3 situation, it might be
+known that normally Alice and Bob will co-sign, while Carol only signs
+if one of the others is unavailable.  For this set of circumstances, the
+primary keys can use a multisignature for the taproot keypath spend (e.g.
+between Alice and Bob) and the additional outcomes (Alice and Carol, or
+Bob and Carol) can use multisignatures with the `OP_CHECKSIG` opcode in
+separate branches of a [tapscript][topic tapscript].
```

Comment:
maybe something like s/separate branches of a tapscript./separate branches of a tree of tapscripts.

edit: I think I see what you mean. Using a single tapscript with OP_CHECKSIGs instead of 2 tapscript leaves in the taproot tree. If so maybe the word branches through me off.

Reply:
Agree with suggestion, will fix.  FYI, I was thinking of two tapscript leaves.  The whole thing is:

       A+B
       /  \
    A+C   B+C